postgres_exporter/common/pg11/setup.sql

```diff
@@ -31,7 +32,8 @@ ALTER TABLE monitor.pgbackrest_info SET (autovacuum_analyze_scale_factor = 0, au
 DROP FUNCTION IF EXISTS monitor.pgbackrest_info(); -- old version from 2.3
 DROP FUNCTION IF EXISTS monitor.pgbackrest_info(int);
 CREATE OR REPLACE FUNCTION monitor.pgbackrest_info(p_throttle_minutes int DEFAULT 10) RETURNS SETOF monitor.pgbackrest_info
-    LANGUAGE plpgsql SECURITY DEFINER
+    LANGUAGE plpgsql
```

Comment:
Is the loss of `SECURITY DEFINER` here intentional?

Reply:
Yes, PG11+ added `pg_execute_server_program`. That has been granted to the ccp_monitoring user now, so definer is no longer needed.